crates/flox/doc/flox-run.md

```diff
@@ -11,21 +11,66 @@ flox-run - run app from current project
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] run [ `<options>` ] [ -- [ `<command args>` ... ] ]
+flox [ `<general-options>` ] run [ `<run-options>` ] [ -- [ `<command args>` ... ] ]
 
 # DESCRIPTION
 
 Run flake application from the requested package (or "installable").
```

Comment:
Nit(optional): still think this would be a bit more clear, because I'm not sure what the requested package is
```suggestion
Run flake application (or "installable").
```